features/pom.xml

```diff
@@ -109,11 +109,11 @@
         </cxf.geronimo.transaction.version>
         <cxf.guava.version>${guava.version}</cxf.guava.version>
         <cxf.hazelcast.version>3.9.2</cxf.hazelcast.version>
-        <cxf.hibernate.validator.version>5.2.3.Final</cxf.hibernate.validator.version>
+        <cxf.hibernate.validator.version>5.3.6.Final</cxf.hibernate.validator.version>
         <cxf.httpcomponents.asyncclient.version>4.1.3</cxf.httpcomponents.asyncclient.version>
         <cxf.httpcomponents.core.version>4.4.9</cxf.httpcomponents.core.version>
         <cxf.james.mim4j.version>0.7.2</cxf.james.mim4j.version>
-        <cxf.jasypt.bundle.version>1.9.0_1</cxf.jasypt.bundle.version>
+        <cxf.jasypt.bundle.version>1.9.2_1</cxf.jasypt.bundle.version>
```

Comment:
❗️ These changes should be documented in the comments above to keep record of what changes from the CXF defaults.